static/app/components/replays/replayPlayer.tsx

```diff
@@ -117,13 +128,26 @@ function BasePlayerRoot({className, isPreview = false}: Props) {
   }, [windowDimensions, videoDimensions]);
 
   return (
-    <NegativeSpaceContainer ref={windowEl} className="sentry-block">
-      <div ref={viewEl} className={className} />
-      {fastForwardSpeed ? <PositionedFastForward speed={fastForwardSpeed} /> : null}
-      {isBuffering ? <PositionedBuffering /> : null}
-      {isPreview ? null : <PlayerDOMAlert />}
-      {isFetching ? <PositionedLoadingIndicator /> : null}
-    </NegativeSpaceContainer>
+    <Fragment>
+      {isFinished && overlayText && (
+        <Overlay>
+          <OverlayInnerWrapper>
+            <UpNext>{t('Up Next')}</UpNext>
+            <OverlayText>{overlayText}</OverlayText>
+            <Button onClick={onClickNextReplay} icon={<IconPlay size="md" />}>
+              {t('Play Now')}
+            </Button>
+          </OverlayInnerWrapper>
+        </Overlay>
+      )}
```

Comment:
This should be inside the NegativeSpaceContainer shouldn't it? then we don't need to change the css on that thing..

all the other bits are `position: absolute` and children, idk why this is different.